### PR TITLE
net-im/qtox: bump to v1.11.0

### PR DESCRIPTION
**Pull request by zetok**
Package-Manager: Portage-2.3.5, Repoman-2.3.1

### CODE REVIEW

**Comment by mgorny** [comment]
@zetok, is that your (alternate) e-mail?

**Comment by zetok** [comment]
@mgorny yep.

File: net-im/qtox/qtox-1.10.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	X? ( x11-libs/libX11
+		x11-libs/libXScrnSaver )
+"
+DEPEND="${RDEPEND}
+	$(unpacker_src_uri_depends)
+	dev-qt/linguist-tools:5
+	virtual/pkgconfig
+"
+
+pkg_pretend() {
```
Comment (mgorny) [inline net-im/qtox/qtox-1.10.1.ebuild]: I think you can drop it at this point. But it's not a biggie.

File: net-im/qtox/qtox-1.10.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+				eerror "You need at least sys-devel/gcc-4.8.3"
+				die "You need at least sys-devel/gcc-4.8.3"
+			fi
+		fi
+	fi
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DENABLE_STATUSNOTIFIER=$(usex gtk True False)
```
Comment (mgorny) [inline net-im/qtox/qtox-1.10.1.ebuild]: Is the build system broken enough to require explicit True/False? Normally CMake is equally happy with our yes/no.

**Comment by mgorny** [comment]
@zetok, ping. Could you answer those two questions?

**Comment by soredake** [comment]
Ping.

**Comment by zetok** [comment]
I've removed not needed `pkg_pretend`, `True False`, old ebuilds and bumped version to 1.11.0.

**Comment by mgorny** [comment]
I've merged it since there are no regressions but please fix the following QA warnings:

```
 * QA Notice: new icons were found installed but GTK+ icon cache
 * has not been updated:
 *   /usr/share/icons/hicolor/128x128/apps/qtox.png
 *   /usr/share/icons/hicolor/16x16/apps/qtox.png
 *   /usr/share/icons/hicolor/192x192/apps/qtox.png
 *   /usr/share/icons/hicolor/22x22/apps/qtox.png
 *   /usr/share/icons/hicolor/24x24/apps/qtox.png
 *   /usr/share/icons/hicolor/256x256/apps/qtox.png
 *   /usr/share/icons/hicolor/32x32/apps/qtox.png
 *   /usr/share/icons/hicolor/36x36/apps/qtox.png
 *   /usr/share/icons/hicolor/48x48/apps/qtox.png
 *   /usr/share/icons/hicolor/64x64/apps/qtox.png
 *   /usr/share/icons/hicolor/72x72/apps/qtox.png
 *   /usr/share/icons/hicolor/96x96/apps/qtox.png
 *   /usr/share/icons/hicolor/scalable/apps/qtox.svg
 *   /usr/share/icons/hicolor/512x512/apps/qtox.png
 *   /usr/share/icons/hicolor/14x14/apps/qtox.png
 * Please make sure to call gnome2_icon_cache_update()
 * in pkg_postinst() and pkg_postrm() phases of appropriate pkgs.
 * QA Notice: .desktop files with MimeType= were found installed
 * but desktop mimeinfo cache has not been updated:
 *   /usr/share/applications/qtox.desktop
 * Please make sure to call xdg_desktop_database_update()
 * in pkg_postinst() and pkg_postrm() phases of appropriate pkgs.

**Pull request closed without merging**